### PR TITLE
I18n sync Down & Out 01/04

### PR DESCRIPTION
**Pull request by deploy-code-org**
# I18n Sync Down & Out

This PR contains all changes to translations made since the last sync (usually a week).

## To Review:

Look through the changes in each of the commits and verify that all changes are expected. Note that **it is not necessary to review every single "dashboard i18n update" commit.** These commits are batched by language to make it easier to review just a couple commits in this category.

### Things to Note:

- We expect translation changes to come in on a regular basis; new strings will be translated, existing translations will be updated, and source string removals will result in translations for those strings also being removed.

- We expect translations to arrive in large batches, so don't worry if you see a large group of changes in a specific language.

- We _do not_ usually expect to see a large number of formatting changes, or batches of changes that apply to strings in every single language. If you see this, it usually means either that a code change went in or that something broke.

## Known Issues:

- If the CI unit tests fail at the `grunt messages:all` step, they will typically fail with a message like:

  ```
  [2019-10-04 02:38:57] Running "messages:all" (messages) task
  [2019-10-04 02:39:00] Warning: Error processing localization file i18n/common/zh_tw.json: SyntaxError: Expected [0-9a-zA-Z$_] but "\u8A0A" found. Use --force to continue. 
  ```

  This usually means that a translator has translated one of the [MessageFormat variables](https://messageformat.github.io/messageformat/guide/#variables). In this case, you should find and fix the offending translation in crowdin, and then manually update the associated strings in `i18n/locales` and `apps/i18n`.

  See https://github.com/code-dot-org/code-dot-org/pull/31098/commits/ab3e5d1d740869a880f2e9da5cf0a4348886915e and https://github.com/code-dot-org/code-dot-org/pull/31098/commits/0fca37658cacaaff55d9adfc9d04c40b37cc5e2d for an example.

## To Deploy:

Once one or two people have reviewed and approved this change and the tests are passing, ship it!

It's not necessary to wait for everyone on the team to review.

### CODE REVIEW

**Comment by wilkie** [comment]
image

File: apps/i18n/applab/my_mm.json [hunk 1]
```diff
@@ -1,14 +1,82 @@
...
+  "dropletBlock_turnRight_param0_description": "ညာဘက်ထောင့်သို့ ရွှေ့ရန်",
+  "dropletBlock_turnTo_description": "လိပ်၏လမ်းကြောင်းကို တိကျသေချာသောထောင့်အနေအထားသို့ပြောင်းပါ။ ၀ သည် အပေါ်ဘက်၊ ၉၀ သည် ညာဘက်၊ ၁၈၀ သည် အောက်ဘက်၊ ၂၇၀ သည် ညာဘက်ဖြစ်ပါသည်။",
+  "dropletBlock_turnTo_param0_description": "လိပ်၏ဦးတည်ချက်သတ်မှတ်ရန် ထောင့်အသစ်။",
+  "dropletBlock_updateRecord_description": "အပ်လပ်(ဘ်)၏ အချက်အလက်ကို အသုံးပြု၍ ဇယားကွက်တွင်ပေးထားသောမှတ်တမ်းကို နာမည်အသစ်ဖြင့် အဆင့်မြင့်ပါ။ မှတ်တမ်းသည် ၎င်း၏ အိုင်ဒီအကွက်ဖြင့် ထူးခြာစွာခွဲထွက်နေရပါမည်။  ခေါ်ဆိုမှုပြီးဆုံးပါက ပြန်လည်ခေါ်ဆိုခြင်းလုပ်ဆောင်ချက်ကို ခေါ်ပြီးပြီဖြစ်ပါသည်။",
+  "dropletBlock_updateRecord_param2_description": "အဆင့်မြင့်တင်သောမှတ်တမ်း()ပြီးဆုံးသောအခါတွင် မတူညီစွာပြန်ခေါ်သည့်လုပ်ဆောင်ချက်တစ်ခု။",
+  "dropletBlock_write_description": "သတ်မှတ်ထားသော စာသားကို စာရွက်စာတမ်း၏ အောက်ခြေတွင် ထပ်မံ၍ပေါင်းထည့်ပါ။ စာသားကိုလည်း HTML ပုံစံအဖြစ်ပြုလုပ်နိုင်သည်။",
+  "dropletBlock_write_param0_description": "သင်ထည့်သွင်းလိုသော စာသား သို့မဟုတ် HTMLသင့်ဆော့ဖ်ဝဲလ်၏အောက်ခြေတွင် ထပ်မံ၍ထည့်သွင်းသည်။",
+  "foundDevices": "အမှတ်စဉ်တွင် အသုံးပြုနိုင်သော စက်ကို ရှာမတွေ့ပါ။\nတွေ့ရှိသော စက်များ- {စက် စာရင်း}",
```
Comment (wilkie) [inline apps/i18n/applab/my_mm.json:31]: Indeed.

Comment (wilkie) [reply apps/i18n/applab/my_mm.json:31]: I think there are redaction/restoration issues also within this translation.

Comment (daynew) [reply apps/i18n/applab/my_mm.json:31]: Yeah a lot of the Burmese translations had issues. I went through the translators history and fixed them all. I will do another down+out to get the updates. Thanks for pointing that out.

**Pull request closed without merging**